app/views/omni_nav_v2/_global_nav.html.erb

```diff
@@ -58,6 +57,18 @@
           </ul>
         </div>
       </li>
+      <li class="primary-link">
+        <a>Research</a>
+        <div class="global-nav-dropdown">
+          <ul>
+            <li><a class="icon icon-medal" href="research/sponsored-projects-services/index.aspx">Sponsored Projects Services</a></li>
+            <li><a class="icon icon-clipboard5" href="research/integrity/index.aspx">Research Integrity</a></li>
+            <li><a class="icon icon-library4" href="research/institutes-and-centers/index.aspx">Institutes and Centers</a></li>
+            <li><a class="icon icon-lamp8" href="research/office-undergraduate-research-creative-activity/index.aspx">Undergraduate Research and Creative Activity</a></li>
+            <li><a class="icon icon-microscope" href="research/graduate-research/index.aspx">Graduate Research Support</a></li>
+          </ul>
+        </div>
+      </li>
```

Comment:
Was this change request added to the story by a product owner?

Reply:
No, it just didn't reflect the either one of the actual global navs and I wanted the text to at least match the screenshots that were reported by the users for this text wrapping issue.

Reply:
Oh, I see. This is just the Rails app sample. Disregard my original comment.

Reply:
@kaylaziegler Meagan Sawyer (new person in UA) would be a good person to ask to test this. She was seeing the odd rendering of the omniNav menu items where the initial letter was a superscript. She didn't uninstall the Minion  Pro font like the other person I'd asked to try that so would still see it broken on prod and fixed on dev. You should email her (or Ross) to ask her to test for you.

Reply:
@mandythomas Thank you for letting me know about Megan, I had her test the staging link that I sent in the Review Request email and it fixed the problem for her! very exciting